gui/probe.lua

```diff
@@ -0,0 +1,71 @@
+-- Shows the info given by `probe` in a friendly display
+
+local gui = require('gui')
+local guidm = require('gui.dwarfmode')
+local widgets = require('gui.widgets')
+local overlay = require('plugins.overlay')
+
+Probe = defclass(Probe, widgets.Window)
+Probe.ATTRS{
+    frame = {
+        w = 40,
+        h = 45,
+        r = 2,
+        t = 18,
+    },
+    resizable=true,
+    frame_title='Probe',
+}
+
+function Probe:init()
+    self:addviews{
+        widgets.ToggleHotkeyLabel{
+            view_id='lock',
+            frame={t=0, l=0},
+            key='CUSTOM_CTRL_F',
+            label='Lock on tile:',
+            initial_option=false,
+        },
+        widgets.Label{
+            view_id='report',
+            frame={t=2, l=0},
+        },
+    }
+end
+
+function Probe:onRenderBody()
+    if self.subviews.lock:getOptionValue() or self:getMouseFramePos() then return end
+    guidm.setCursorPos(dfhack.gui.getMousePos())
```

Comment:
I'm not a fan of this approach. I'll go update `probe` to accept the cursor position on the commandline.

Reply:
edit: `probe` is now updated to take a cursor position. you can call `probe --cursor=x,y,z`

Reply:
I'll rebase and do that then.